ibis/backends/tests/test_generic.py

```diff
@@ -163,3 +164,26 @@ def test_case_where(backend, alltypes, df):
     expected['new_col'] = expected['new_col']
 
     backend.assert_frame_equal(result, expected)
+
+
+@pytest.mark.xfail_unsupported
+@pytest.mark.skip_backends(['postgres'])
+def test_select_filter_mutate(backend, alltypes, df):
+    t = alltypes
```

Comment:
can you add some comments here and the issue refernce explaining what this test is testing.

Reply:
Added